doc/conf.py

```diff
@@ -80,7 +80,7 @@
 year = datetime.date.today().year
 project = "Harmonica"
 copyright = "2018-{}, The Harmonica Developers".format(year)
-if len(full_version.split("+")) > 1 or full_version == "unknown":
+if "dev" in full_version:
     version = "dev"
 else:
     version = full_version
```

Comment:
Now that we are using `no-local-version` by default, we can simplify how we detect if we are running a development version or a released one. Do you like it @leouieda ?